src/App.tsx

```diff
@@ -1,23 +1,92 @@
 import "./App.css";
-
+import React, { useState } from 'react';
 import { taskList } from "./taskList";
 
+let taskListCopy = taskList;
+
+// Create a list with additional field to mark deleted task
+  let taskListExt: any[] = [];
+  for (let i=0 ; i < taskListCopy.length; i++){
+    taskListExt.push(
+      {
+        'id'  : taskListCopy[i].id,
+        'name': taskListCopy[i].name,
+        'del' : '' 
+      }
+    )
+  }
+
+
+//
+
 function TaskShow() {
+  let [tasks, setTasks ] = useState(taskListExt);
+  let [newTask, setNewTask] = useState('');
```

Comment:
There aren't changed later on, so use `const` instead of `let`

Reply:
Also, since you used the convention `taskListExt`, this should be `taskList` instead of `tasks` for consistency. Or you can change the other one to `tasksExt`